separatemodels/src/test/java/pl/altkom/asc/lab/cqrs/intro/separatemodels/testdatabuilders/OffersTestDataBuilder.java

```diff
@@ -11,7 +9,7 @@
 import java.util.UUID;
 
 public class OffersTestDataBuilder {
-    static Offer standardOneYearOCOfferValidUntil(LocalDate validityEnd) {
+    static Offer standardOneYearOCOfferValidUntil(LocalDate validityEnd, Person customer, Car car) {
```

Comment:
After adding these two arguments the method name doesn't make sense anymore (the "ValidUntil" part referred strictly to the first argument). Maybe this method should return `OffersTestDataBuilder` with additional methods allowing to set also customer and car?